### PR TITLE
Add a pop up javascript menu

### PR DESCRIPTION
**Pull request by inspecta**
- Add a HTML and styles for the menu list in the mobile version
- Add a separate JavaScript file to update DOM elements 
- Update my Linters.yml to check for JavaScript syntax errors

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+hamburger.addEventListener('click', displayMenuList);
+menuClose.addEventListener('click', closeMenuList);
+
+document.querySelectorAll('.menu-item-list').forEach((n) => {
+  n.addEventListener('click', () => {
+    menuOverlay.style.display = 'none';
+    hamburger.style.display = 'block';
+  });
+});
```
Comment (sja-thedude) [inline main.js]: - [ ] Kindly display the hamburger icon only for the mobile menu. Remove the hamburger icon from the desktop version as stated in the attached loom video.
- [ ] Kindly follow the Figma design for the mobile popup menu. Use the backdrop with the background colour: `background: #6070FF;` to make it as close to the design as possible as shown in the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @inspecta, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional readme
✔️ Correct gitflow

## Required 
- [ ] Kindly make sure to view this loom video to make changes [here](https://www.loom.com/share/cbed097b1a0b482b81cd659477e678bc).
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (inspecta) [reply main.js]: Thanks @sja-thedude for the review and for the specific directions 👏

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @inspecta,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request**

You have submitted a project, that _**You have sent the PR with no linters installation, please check the linters are passing and running before submitting again.**_  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by roshan-bajgain — APPROVED** [review]
Hi @inspecta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**